galsim/wcs.py

```diff
@@ -546,18 +546,19 @@ def affine(self, image_pos=None, world_pos=None, color=None):
                 world_pos = self.toWorld(image_pos, color=color)
             return jac.withOrigin(image_pos, world_pos, color=color)
 
-    def withOrigin(self, origin, world_origin=None, color=None):
-        """Recenter the current WCS function at a new origin location, returning the new WCS.
+    def shiftOrigin(self, origin, world_origin=None, color=None):
+        """Shift the origin of the current WCS function, returning the new WCS.
 
-        This function creates a new WCS instance (always a non-local WCS) that treats
-        the origin position the same way the current WCS treats (x,y) = (0,0).
+        This function creates a new WCS instance (always a non-local WCS) that shifts the
+        origin by the given amount.  In other words, it treats the image position ``origin``
+        the same way the current WCS treats (x,y) = (0,0).
```

Comment:
The last statement in this paragraph confuses me. I think it is technically correct, but the language throws me. IDK if there is anything to do about this though and I don't have a good suggestion.

Reply:
I guess the point I was trying to make is that the "origin" of the current wcs doesn't have to be (0,0).  It is if it's local, but not necessarily otherwise.  So whatever real position on the image is currently called (0,0), the new wcs will call that position `origin`, the argument being passed to this function.  I'm not sure what would be a better sentence to explain that, but if you can think of something that would make that clearer, I'm game to change it.

Reply:
Yeah I’ve got nothing. I’d say merge the pr and I can make another later to change things if I think I can say it more clearly.

Reply:
OK.  :)